### ISSUE TITLE
Stripe.Net nuget is outdated

### ISSUE THREAD

**Issue by Theoistic** [eralston/StripeEntities#1]
The nuget package that is used is outdated and no longer matches the stripe.net package or source, the stripe.net package was updated in accordance to api changes. StripeEntities targets stripe.net 2.x but current version is at 4.x

**Comment by eralston** [comment]
Theodor,

Thank you for finding the project and telling me.

I actually just bumped the dependency to 4.2 last night with this change:

https://github.com/eralston/StripeEntities/commit/0ad61e29af2140be58fbf9bcffb181210ea89c4a

I just need to redeploy to nuget package after verifying the supporting the
changes to how stripe.net is consumed still work. Please expect a release
this week.

Thank you,
Erik

**Comment by eralston** [comment]
Spero,

I have published a new version of StripeEntities to NuGet, version 2.0.0:

https://www.nuget.org/packages/StripeEntities/2.0.0

This release includes full XML documentation. It also benefits from the set of unit tests I just published in the project - meaning I found a bug where unsubscribe never worked as originally coded!

I also renamed all of the interfaces and base classes for better consistency with the Stripe.Net API (something I had been thinking about for quite a while), meaning this will have a breaking change with any existing code consuming the API names. The transition should be straightforward.

Thank you for grabbing my attention on this so I could really improve the library and good luck!

Thank you very much,
Erik
